### PR TITLE
Update issues form

### PR DESCRIPTION
**Pull request by bsousaa**
**- What I did**
Update the repository issues form to support bug and features requests.

Additionally, added links to other resources to guide users across support documentation and forums.

Signed-off-by: Bruno Sousa <bruno.sousa@docker.com>

**- How I did it**
Using the latest features available on GitHub to manage [issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) and using the same template from the [Moby Project](https://github.com/moby/moby/).

**- How to verify it**
Create a new issue after the PR is merged and confirm you are given the options to create either a bug or a feature request.

**- Description for the changelog**
Update the repository issues form to support bug and features requests.

**- A picture of a cute animal (not mandatory but encouraged)**
9eec8720-13b3-41fd-8068-74f6fcbd3e4f
Gaspar :heart:

### CODE REVIEW

**Comment by thaJeztah** [comment]
I saw a discussion about this in the containerd project; looks like github may have started adding rate-limits, which cause vanity URLs to fail with go modules (if `GOPROXY=direct`) 😞 

```
#14 312.3 go: github.com/containerd/containerd@v1.6.4 requires
#14 312.3 	gopkg.in/yaml.v2@v2.4.0: unrecognized import path "gopkg.in/yaml.v2": reading https://gopkg.in/yaml.v2?go-get=1: 502 Bad Gateway
#14 312.3 	server response: Cannot obtain refs from GitHub: error reading from GitHub: net/http: request canceled (Client.Timeout exceeded while reading body)
#14 ERROR: executor failed running [/bin/sh -c set -e
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+          Live Restore Enabled: true
+    validations:
+      required: true
+  - type: textarea
+    id: additional
+    attributes:
+      label: Additional Info
+      description: Additional info you want to provide such as logs, system info, environment, etc.
+    validations:
+      required: false
```
Comment (thaJeztah) [inline .github/ISSUE_TEMPLATE/bug_report.yml:146]: Small nit, but may cause issues when we update these files; could you fix the missing newlines at the end of these files?

**Review by thaJeztah — COMMENTED** [review]
Thanks! just a quick comment; haven't looked very closely yet 😅

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (thaJeztah) [reply .github/ISSUE_TEMPLATE/bug_report.yml:146]: ^^ also for the other files

Comment (bsousaa) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Thanks! Newlines added.

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
Let's get this one in; after this we can also consider adding a link to the buildx and/or buildkit repository for build-related issues (/cc @crazy-max wdyt?)

**Comment by crazy-max** [comment]
Yes sounds good @thaJeztah

**Pull request merged**